docs/source/guides/migration.rst

```diff
@@ -2,9 +2,9 @@
 1.0 Migration Guide
 ===================
 
-BentoML version 1.0.0 APIs are backward incompatible with version 0.13.1. However, most of the common 
-functionality can be achieved with the new version. We will guide and demonstrate the migration by 
-transforming the `quickstart <https://github.com/bentoml/BentoML/tree/main/examples/quickstart>`_ gallery project 
+BentoML version 1.0.0 APIs are backward incompatible with version 0.13.1. However, most of the common
+functionality can be achieved with the new version. We will guide and demonstrate the migration by
+transforming the `quickstart <https://github.com/bentoml/BentoML/tree/main/examples/quickstart>`_ gallery project
```

Comment:
Will probably need a reflow. (I know this is a just a whitespace change, but we may as well make these changes)

```suggestion
BentoML version 1.0.0 APIs are backward incompatible with version 0.13.1. However, most of the
functionality is maintained in the new version. We will demonstrate this by
migrating the `quickstart <https://github.com/bentoml/BentoML/tree/main/examples/quickstart>`_ gallery project.
```